scripts/apps/web-publisher/services/publisher-factory.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @ngdoc service
+ * @module superdesk.apps.web_publisher
+ * @name PublisherFactory
+ * @requires api/superdesk.apps.web_publisher/service/PubAPIFactory pubapi
+ * @description Publisher service
+ */
+PublisherFactory.$inject = ['pubapi'];
+export function PublisherFactory(pubapi) {
+
+    class Publisher {
+
+        /**
+         * @ngdoc method
+         * @name PublisherFactory#setTenant
+         * @param {String} tenant
+         * @return this
```

Comment:
It's `@returns`, not `@return`. And you need to specify a type, so maybe `@returns {Object} the instance for chaining`.